dependencies.gradle

```diff
@@ -42,6 +42,7 @@ dependencies {
     api("codechicken:codechickenlib:3.2.3.358")
     api("com.cleanroommc:modularui:2.5.0-rc2") { transitive = false }
     api("com.cleanroommc:groovyscript:1.2.0-hotfix1") { transitive = false }
+    api("curse.maven:inventory-bogosorter-632327:4951607-deobf-4951608-sources-4951609")
```

Comment:
This is added as a dep, but nothing is done with it in this PR that I can see, do we still need it?

Reply:
this is required due to ModularContainer implementing a bogosorter interface, which is loaded from `GregTechGuiTransferHandler`